pkg/snet/addrutil/addrutil.go

```diff
@@ -39,7 +40,7 @@ type Pather struct {
 }
 
 // GetPath computes the remote address with a path based on the provided segment.
```

Comment:
Also make it explicit here that the returned address if discovery information exist will be a random entry, which is not deterministic.

Reply:
Done.